### PR TITLE
Release 0.23.0

### PR DESCRIPTION
**Pull request by VilleMiekkoja**
## Description

v0.23.0 release

### CODE REVIEW

File: site/docs/about/new.mdx [hunk 1]
```diff
@@ -13,7 +13,12 @@ import Link from "../../src/components/Link";
     Here you will find summarized patch notes of major releases of HDS. For full patch notes for each release, please refer to the <Link href="https://github.com/City-of-Helsinki/helsinki-design-system/releases" external>GitHub releases</Link>.
 </LargeParagraph>
 
-<p><strong>0.22.0 (beta)</strong> - <em>3.3.2021</em> - <Link href="https://github.com/City-of-Helsinki/helsinki-design-system/releases/tag/v0.22.0" external>Release notes</Link></p>
+<p><strong>0.23.0 (beta)</strong> - <em>12.3.2021</em> - <Link href="https://github.com/City-of-Helsinki/helsinki-design-system/releases/tag/v0.22.0" external>Release notes</Link></p>
```
Comment (minevala) [inline site/docs/about/new.mdx:16]: Should the link point to version v0.23.0?

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.23.0] - March 3, 2021
```
Comment (minevala) [inline CHANGELOG.md:9]: Today, probably 😁

Comment (VilleMiekkoja) [reply CHANGELOG.md]: Fixed

File: site/docs/about/new.mdx (hunk 1 above)
Comment (VilleMiekkoja) [reply site/docs/about/new.mdx]: Fixed

**Review by minevala — APPROVED** [review]
Good job 👍

**Review by laurakarhu — APPROVED** [review]
Ready to roll 👍

**Pull request merged**